runner/src/server/plugins/engine/page-controllers/DefaultPageController.ts

```diff
@@ -52,4 +54,28 @@ export class DefaultPageController extends PageController {
 
         };
     }
+
+    overrideGetRelevantPages = (state: FormSubmissionState) => {
```

Comment:
Can we give this a better name please that describes why it's overriden, and add some comments about what it does differently? Thanks

Reply:
Fixed, Added a comment